main.md

```diff
@@ -523,14 +521,16 @@ Again, the SD-JWT-R follows the same structure as the `_sd` in the SD-JWT. For E
 }
 ```
 
-The SD-JWT-R MAY contain further claims, for example, to ensure a binding
-to a concrete transaction (in the example the `nonce` and `aud` claims).
+When the holder sends SD-JWT-R to the Verifier, it MUST be a base64url-encoded 
```

Comment:
"base64url-encode JWS" is a confusing description.  If the JWS uses the Compact Serialization, its parts are already base64url-encoded, therefore, one wouldn't re-encoded the JWS (which the current wording seems to imply).  Whereas, if it uses the JSON Serialization, then yes, one would need to base64url-encode it.  This should be clarified in the description.